leakcanary-sample/src/main/AndroidManifest.xml

```diff
@@ -18,8 +18,12 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.leakcanary"
     >
+
+  <uses-feature android:name="android.software.leanback" android:required="false"/>
```

Comment:
This shouldn't be needed, since it's already done in the LeakCanary project, right?